modules/renderer.js

```diff
@@ -9,29 +9,31 @@
  * @param {string} cssSelector
  */
 export function renderer_start(cssSelector) {
-    var draw = SVG(cssSelector).size(window.innerWidth, window.innerHeight)
+    var draw = SVG(cssSelector).size(window.innerWidth, window.innerHeight);
     //var rect = draw.rect(100, 100).attr({ fill: '#f06' })
     //centerCircle(draw)
     //drawNumbers_HeeboStroke(draw)
     var fnt = new Heebo();
-    var disp = fnt.drawNumbers(draw,(window.innerWidth/4),(window.innerHeight/3),'20:59:59')
-	console.log(disp)
+    var disp = fnt.drawNumbers(draw,(window.innerWidth/4),(window.innerHeight/3),'20:59:59');
+    console.log(disp);
     //fnt.transitionNumbers(draw, (window.innerWidth/4), (window.innerHeight/3), disp, curTimeString())
-	setInterval(function() { updateClock(fnt, draw, disp)}, 1000)
+    setInterval(function() { updateClock(fnt, draw, disp); }, 1000);
 }
 
 function updateClock(font, draw, disp) {
-	font.transitionNumbers(draw, (window.innerWidth/4), (window.innerHeight/3), disp, curTimeString())
+    font.transitionNumbers(draw, (window.innerWidth/4), (window.innerHeight/3), disp, curTimeString());
 }
 
 /**
  * centerCircle(SVG: canvas)
  * Reference function to draw a circle at the center of the SVG.
  * @param {SVG} canvas 
  */
+/*
 function centerCircle(canvas) {
-    let circle = canvas.circle(200).attr({fill: '#a06', cx: window.innerWidth/2, cy: window.innerHeight/2 })
+    let circle = canvas.circle(200).attr({fill: '#a06', cx: window.innerWidth/2, cy: window.innerHeight/2 });
```

Comment:
please give a more descriptive name. purpleCircle?

Reply:
Well, when I wrote the function, all I wanted was a circle in the center of the screen... maybe drawCenterCircle?

Not a huge deal. The color probably doesn't need to be in the name.